tests/simple/testdata/basic.textproto

```diff
@@ -133,6 +143,36 @@ section {
     expr: "true"
     value: { bool_value: true }
   }
+  test {
+    name: "self_eval_int_hex"
+    expr: "0x55555555"
+    value: { int64_value: 1431655765 }
+  }
+  test {
+    name: "self_eval_int_hex_negative"
+    expr: "-0x55555555"
+    value: { int64_value: -1431655765 }
+  }
+  test {
+    name: "self_eval_uint_hex"
+    expr: "0x55555555u"
+    value: { uint64_value: 1431655765 }
+  }
+  test {
+    name: "self_eval_unicode_escape_four"
+    expr: '"\\u270c"'
+    value: { string_value: "\xe2\x9c\x8c" }
+  }
+  test {
```

Comment:
Would you mind adding a test to see if code points above 0x10FFFF error?

Reply:
There is no nice way to do that, given expression a code point that high should fail to parse.

Reply:
Fair point.